File: Examples/DotNET/CSharp/Folders/CreateFolder.cs
using System;
using Com.Aspose.Storage.Api;
using Com.Aspose.Storage.Model;

namespace Folders
{
    class CreateFolder
    {
        public static void Run()
        {
            // ExStart:1           
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            string Path = "testfolder";
            string storage = null;
            string destStorage = null;   
            try
            {
                // Invoke Aspose.Storage Cloud SDK API to create folder
                storageApi.PutCreateFolder(Path, storage, destStorage);
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/PHP/Folders/CreateFolder.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Storage\StorageApi;
use Aspose\Storage\AsposeApp;

class Folder {

	public $storageApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->storageApi = new StorageApi();
	}

	public function createFolder() {
		
        $result = $this->storageApi->PutCreateFolder($Path="testfolder", $storage = null, $destStorage = null);
        print_r($result);
	}
}

$folder = new Folder();
$folder->createFolder();

?>
//ExEnd:
